### ISSUE TITLE
Oracle Linux 7.6: scylla_setup: local variable 'res' referenced before assignment

### ISSUE THREAD

**Issue by amoskong** [scylladb/scylla#4842]
*Installation details*
Scylla version (or git commit hash): 3.1.0.rc3-0.20190812.cf4c238b2
Cluster size: 1
OS (RHEL/CentOS/Ubuntu/AWS AMI):  Oracle Linux 7.6

```
[root@localhost ~]# cat /etc/redhat-release 
Red Hat Enterprise Linux Server release 7.6 (Maipo)
[root@localhost ~]# cat /etc/os-release 
NAME="Oracle Linux Server"
VERSION="7.6"
ID="ol"
VARIANT="Server"
VARIANT_ID="server"
VERSION_ID="7.6"
PRETTY_NAME="Oracle Linux Server 7.6"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:oracle:linux:7:6:server"
HOME_URL="https://linux.oracle.com/"
BUG_REPORT_URL="https://bugzilla.oracle.com/"

ORACLE_BUGZILLA_PRODUCT="Oracle Linux 7"
ORACLE_BUGZILLA_PRODUCT_VERSION=7.6
ORACLE_SUPPORT_PRODUCT="Oracle Linux"
ORACLE_SUPPORT_PRODUCT_VERSION=7.6
```

```
2019-08-12 22:21:57,719 process          L0333 INFO | Running '/usr/lib/scylla/scylla_setup --nic ens2 --disks /dev/sdb'
2019-08-12 22:22:38,068 process          L0420 DEBUG| [stdout] WARN  2019-08-12 22:22:38,059 [shard 0] iotune - Available space on filesystem at /var/tmp/mnt: 124 MB: is less than recommended: 10 GB
2019-08-12 22:22:38,068 process          L0420 DEBUG| [stdout] INFO  2019-08-12 22:22:38,059 [shard 0] iotune - /var/tmp/mnt passed sanity checks
2019-08-12 22:22:38,627 process          L0420 DEBUG| [stdout] This is a supported kernel version.
2019-08-12 22:22:38,636 process          L0420 DEBUG| [stderr] Traceback (most recent call last):
2019-08-12 22:22:38,636 process          L0420 DEBUG| [stderr]   File "/usr/lib/scylla/libexec/scylla_setup", line 241, in <module>
2019-08-12 22:22:38,636 process          L0420 DEBUG| [stderr]     do_verify_package('scylla-jmx')
2019-08-12 22:22:38,636 process          L0420 DEBUG| [stderr]   File "/usr/lib/scylla/libexec/scylla_setup", line 98, in do_verify_package
2019-08-12 22:22:38,636 process          L0420 DEBUG| [stderr]     if res != 0:
2019-08-12 22:22:38,636 process          L0420 DEBUG| [stderr] UnboundLocalError: local variable 'res' referenced before assignment
2019-08-12 22:22:38,644 stacktrace       L0036 ERROR| 
2019-08-12 22:22:38,645 stacktrace       L0039 ERROR| Reproduced traceback from: /usr/lib/python2.7/site-packages/avocado/core/test.py:436
2019-08-12 22:22:38,645 stacktrace       L0042 ERROR| Traceback (most recent call last):
2019-08-12 22:22:38,646 stacktrace       L0042 ERROR|   File "/root/avocado/tests/jenkins/slave/workspace/scylla-3.1/oel7.4-rpm-test/label/muninn/scylla-artifacts.py", line 765, in setUp
2019-08-12 22:22:38,646 stacktrace       L0042 ERROR|     self.scylla_setup()
2019-08-12 22:22:38,646 stacktrace       L0042 ERROR|   File "/root/avocado/tests/jenkins/slave/workspace/scylla-3.1/oel7.4-rpm-test/label/muninn/scylla-artifacts.py", line 751, in scylla_setup
2019-08-12 22:22:38,646 stacktrace       L0042 ERROR|     installer.run()
2019-08-12 22:22:38,646 stacktrace       L0042 ERROR|   File "/root/avocado/tests/jenkins/slave/workspace/scylla-3.1/oel7.4-rpm-test/label/muninn/scylla-artifacts.py", line 370, in run
2019-08-12 22:22:38,646 stacktrace       L0042 ERROR|     process.run(setup_cmd, shell=True, verbose=True, timeout=600)
2019-08-12 22:22:38,646 stacktrace       L0042 ERROR|   File "/usr/lib/python2.7/site-packages/avocado/utils/process.py", line 1031, in run
2019-08-12 22:22:38,646 stacktrace       L0042 ERROR|     raise CmdError(cmd, sp.result)
2019-08-12 22:22:38,646 stacktrace       L0042 ERROR| CmdError: Command '/usr/lib/scylla/scylla_setup --nic ens2 --disks /dev/sdb' failed (rc=1)
```

**Comment by amoskong** [comment]
duplicated with https://github.com/scylladb/scylla/issues/4493
We need to backport the fix to branch-3.1

@glommer

**Comment by slivne** [comment]
merged into 3.1 062d43c76eaf1bb5373e0bf25f96d0785800c315
